style.css

```diff
@@ -1,169 +1,293 @@
-html {
-  font-size: 62.5%;
-}
 
-body {
-  margin: 0;
-  color: #fff;
-  font-family: inter, sans-serif;
-  max-width: 100%;
-}
+/* Mobile responsiveness */
+@media screen and (max-width: 768px) {
+  html {
+    font-size: 62.5%;
+  }
 
-nav {
-  display: flex;
-  margin: auto;
-  background-color: #3e3c3c;
-  font-size: 1.5rem;
-  font-weight: bold;
-  align-items: center;
-  justify-content: space-between;
-}
+  * {
+    box-sizing: border-box;
+  }
 
-nav img {
-  margin-right: 2rem;
-}
+  body {
+    margin: auto;
+    font-family: inter, sans-serif;
+  }
 
-@media screen and (max-width: 768px) {
-  #menu {
-    display: none;
+  /* Nav bar styling */
+  #navbar {
+    display: flex;
+    background-color: #3e3c3c;
+    align-items: center;
+    justify-content: space-between;
+    padding: 0 3.5rem 0 2.4rem;
+  }
+
+  nav ul {
+    margin: 0;
   }
 
   #logo {
-    margin-left: 2rem;
-    padding: 2rem 0;
+    width: 10%;
+    height: auto;
+    border-radius: 50%;
   }
 
-  #headline {
-    flex-direction: column;
-    background-position-y: bottom;
+  a {
+    text-decoration: none;
+    color: #fff;
+  }
+
+  .no-style-deco {
+    list-style: none;
+  }
+
+  .white {
+    color: #fff;
+  }
+
+  /* Home section styling */
+  #home {
+    background-image: url(media/Illustration-header.png);
+    background-color: #1c1a19;
+    background-attachment: scroll;
+    background-repeat: no-repeat;
     background-size: 25%;
+    height: 25%;
+    background-position: right bottom;
+    display: flex;
+    flex-direction: column;
+    padding: 12rem 2.4rem;
+  }
+
+  .orange-color {
+    color: #f55800;
+    font-family: 'Crete Round', serif;
+    font-size: 5.6rem;
+    margin: 0;
+    font-weight: 400;
+  }
+
+  #home p {
+    font-weight: 400;
+    font-family: 'poppins', sans-serif;
+    font-size: 1.6rem;
+    margin: 0;
+  }
+
+  #home h2 {
+    font-family: 'roboto', sans-serif;
+    font-weight: 700;
+    font-size: 2.4rem;
+    color: #fff5e1;
+    margin: 1.2rem 0;
+  }
+
+  /* Social media links styling */
+  #social-items {
+    display: flex;
+    align-self: center;
+    margin-top: 1.2rem;
+    padding: 0;
+  }
+
+  .social-item {
+    margin: 0 1.5rem;
+  }
+
+  /* Styling works section */
+  #projects {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 12rem 2.4rem;
+  }
+
+  #work-title {
+    font-family: 'Crete Round', sans-serif;
+    font-size: 4rem;
+    font-weight: 400;
+  }
+
+  .line {
+    border: 0.05rem solid #212121;
+    width: 100%;
+    margin: 2.4rem 0 8rem 0;
+  }
+
+  #work-cover {
+    width: 100%;
+    height: auto;
+    margin-bottom: 2.4rem;
+  }
+
+  .project-item {
+    margin-bottom: 2.4rem;
+    background-image: url("media/Img Placeholder.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+    padding: 7.2rem 1.6rem 0 1.6rem;
+    height: 100vw;
+    width: 100%;
+    position: relative;
   }
```

Comment:
- [ ] Please ensure that the project card styling is aligned with the Figma design as much as possible 💯, currently the tech list is overlapping on the `see project` button.

### Your Design
![image](https://user-images.githubusercontent.com/68030297/228986310-2323fbad-867c-4796-bc32-e994c0ab141c.png)

### Figma Design
![image](https://user-images.githubusercontent.com/68030297/228986442-6155bd01-b65b-405f-9a66-41b6825104ee.png)

### Suggestions:
If you avoid setting width and height property along the way in this project it will be great for the layout since all element gets their own width and height depending on the content 👍🏼 this will help you a lot with the website responsiveness 💪🏼 

- You need to set a padding-bottom the same value as the padding-top, but since the height for `project-item` is set to ` 100vh` which is a fixed height, you won't see any results.

So kindly try deleting the `height` property and try setting a `padding-bottom` with the value `7.2rem` 👍🏼 You will love what you see 😎.